### DISCUSSION TITLE
[Feature] Context searching for shared albums

### DISCUSSION THREAD

**Discussion by Hiers** [immich-app/immich#11973] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I really like how well the context searching works in immich, but I can't use it to the extent that I want.

I'm in a situation where I don't have many pictures myself, but my family has a shared album where we put pictures we want to share with everyone. Most of the time I want to use context searching, I want to use it to search an image in that album with tens of thousands of pictures going back as far as the 90s. Adding shared albums as an option for context search would be very useful for my use case.

### Platform

- [ ] Server
- [x] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2995
